package.json

```diff
@@ -103,6 +97,15 @@
 					"type": "boolean",
 					"description": "Controls whether Rech Batch extension should alert when conflicting batch extensions are detected.",
 					"default": true
+				},
+				"rech.batch.snippetsType": {
+					"type": "string",
+					"default": "Community",
+					"description": "Switch to different snippets (you should use the community configuration if you are not at Rech)",
```

Comment:
I don't know if it is properly to say that the user should use the community setting if he's not at Rech.

User may like Rech's snippets and use it if he can understand portuguese language.

@zLianK @Lucasspaniol What do you think about it?

Reply:
I agree with you, let me change this literal.